src/acf.py

```diff
@@ -122,12 +129,27 @@ def disconnect(self):
         except:
             pass
 
+    def timer_callback(self):
+        self.send_command(self.force)
+        self.initial_force = self.force
+        telem = self.handle_telem()
+        if self.joint_state_pub is not None:
+            msg = JointState()
+            msg.header.stamp = self.get_clock().now().to_msg()
+            msg.header.frame_id = f"{self.i}"
+            msg.name = [self.joint_name]
+            msg.position = [telem.position/1000]
```

Comment:
Might be nice to refactor this division and make it a stand-alone function. If you call it `to_metres(..)`, the name of the function makes intent crystal clear and would also facilitate debugging.

Additionally: consider dividing by `1e3`.

Reply:
Will leave as is for now, since it is a once-off conversion. Have added a comment to clarify.